File: extract.js
const fs = require("fs");
const { execSync } = require("child_process");

// Try using node's built-in modules to handle the zip
try {
  // Check if unzip is available
  execSync("unzip -qq nova-landing.zip", { stdio: "inherit" });
  console.log("Extracted successfully with unzip");
} catch (error) {
  console.log("Unzip failed, trying alternative method...");
  console.log("Error:", error.message);
}

File: extract.py
import zipfile
import os

with zipfile.ZipFile('nova-landing.zip', 'r') as zip_ref:
    zip_ref.extractall('.')
    print("Files extracted successfully!")
    print("Contents:")
    for name in zip_ref.namelist()[:15]:
        print(f"  {name}")
